Base/usr/share/man/man1/cal.md

```diff
@@ -5,73 +5,104 @@ cal - Display a calendar
 ## Synopsis
 
 ```**sh
-$ cal [[[day] month] year]
+$ cal [--monday-starts-week] [--year] [--three-month-view] [[month] year]
 ```
 
 ## Description
 
-This program displays a simple calendar. If no arguments are specified, the current month is displayed with the current day highlighted.
+This program displays a simple calendar. If no arguments are specified, the current month is displayed.
+An overview of a whole year is displayed when a `year` is passed without a `month`.
+
+The current day is always highlighted.
+
+Months and years are specified with numbers. Week starts at Sunday, unless the `--monday-starts-week` option is passed.
+
+## Options
+
+* `-M`, `--monday-starts-week`: Make Monday the first day of the week, as opposed to Sunday - the default.
+
+* `-y`, `--year`: Display an entire year. Can be used to display the current year without specifying which one.
```

Comment:
This wording is slightly confusing, better would be "If no year number is specified, the current year is used as a default".

Reply:
That's not exactly what I'm trying to communicate here, but your description is better than mine - changed.

The intent here was to communicate the sole reason for `-y` to exist - to display the current year. Displaying any other year can be done by just specifying it to `cal`, e.g. `cal 1945` will give you the overview of the year 1945. This makes it troublesome to display the current year quickly, because you have to specify e.g. `cal 2023`. The `-y` option is not useful at all for any other year than the current one.

I'm not sure how to communicate that effectively in the manual, maybe it's too specific.

Reply:
(i guess that the second part was too detailed and could be left out—maybe the full option name should be named `--whole-year` instead? 🤔)
```markdown
* `-y`, `--whole-year`: Display an entire year.
```
still one would probably need to guess what displaying whole/entire year exactly means: maybe a word or two on that?

Reply:
@konradekk I've changed it to say `Display an entire year by laying out months on a grid.`. Do you think that's better? I've been staring at it for too long and now I have no idea

Reply:
I think that's good.